MH4U Dex/ItemSourcesTableViewController.m

```diff
@@ -0,0 +1,112 @@
+//
+//  ItemSourcesTableViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/19/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "ItemSourcesTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "Constants.h"
+#import "CoreDataController.h"
+
+#import "MonsterContainerViewController.h"
+#import "RegionContainerViewController.h"
+
+#import "Area.h"
+#import "AreaDrop.h"
+#import "Drop.h"
+#import "Item.h"
+#import "Monster.h"
+#import "MonsterDrop.h"
+
+#import "ItemSourceTableViewCell.h"
+
+@interface ItemSourcesTableViewController ()
+
+@property (nonatomic, strong) NSArray *sources;
+
+@end
+
+@implementation ItemSourcesTableViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    NSManagedObjectContext *managedObjectContext = [CoreDataController sharedCDController].managedObjectContext;
+    NSError *fetchError = nil;
```

Comment:
I know you just moved this from another file, but: you don't need to explicitly set new variables to nil.